src/Hocon.Configuration/Config.cs

```diff
@@ -27,7 +27,7 @@ public class Config : HoconRoot, ISerializable, IEquatable<Config>
         static Config()
         {
             EmptyValue = new HoconValue(null);
-            EmptyValue.Add(new HoconObject(EmptyValue));
+            EmptyValue.Add(new InternalHoconObject(EmptyValue));
```

Comment:
Old classes still lives in the code, they are all changed from `public` to `internal` and all name collision are resolved by adding `Internal` to the start of the class name.